### PR TITLE
Documentation for materials.py file

### PR DESCRIPTION
**Pull request by GabrielBachmann**

### CODE REVIEW

File: ross/materials.py [hunk 1]
```diff
@@ -115,6 +197,23 @@ def load_data():
 
     @staticmethod
     def use_material(name):
+        """Function to load the materials properties and instantiate a Material Object.
+
+        Parameters
+        ----------
+        name : Material's name.
```
Comment (flaviorangel) [inline ross/materials.py]: In order to keep up with the standard we're using, this should be:

name: string
        Material's name.

File: ross/materials.py [hunk 2]
```diff
@@ -129,6 +228,23 @@ def use_material(name):
 
     @staticmethod
     def remove_material(name):
+        """Function used to delete a saved material.
+
+        Parameters
+        ----------
+        name : Name of Material Object to be deleted.
```
Comment (flaviorangel) [inline ross/materials.py]: Same here:

name: string
Name of Material Object to be deleted.

**Comment by raphaeltimbo** [comment]
@GabrielBachmann, are you working on this PR or did you move the code to a different PR?
If you've moved the code we could close this PR.

**Comment by ross-bott** [comment]
Hi there!
I have marked this issue as stale because it has not had activity for 45 days.
Consider the following options:
- If the issue refers to a large task, break it in smaller issues that can be solved in
less than 45 days;
- Label the issue as `wontfix` or `wontfix for now` and close it.

**Comment by GabrielBachmann** [comment]
#433 substitutes this PR

**Pull request closed without merging**